server_admin/topics/clients/client-policies.adoc

```diff
@@ -0,0 +1,159 @@
+* Sending a token request
+* Sending a token refresh request
+* Sending a token revocation request
+* Sending a token introspection request
+* Sending a userinfo request
+* Sending a logout request with a refresh token
+
+On each event, an executor can work in multiple phases. For example, on creating/updating a client, the executor can modify the client configuration by auto-configure specific client
+settings. After that, the executor validate this configuration in validation phase.
+
+One of several purposes for this executor is to realize security requirements of client conformance profiles like FAPI. To do so, following executors are needed:
+
+* Enforce secure <<_client-credentials,Client Authentication method>> is used for the client
+* Enforce <<_mtls-client-certificate-bound-tokens,Holder-of-key tokens>> are used
+* Enforce <<_proof-key-for-code-exchange,Proof Key for Code Exchange (PKCE)>> is used
+* Enforce secure signature algorithm for <<_client-credentials,Signed JWT client authentication (private-key-jwt)>> is used
+* Enforce HTTPS redirect URI and make sure that configured redirect URI does not contain wildcards
+* Enforce OIDC `request` object satisfying high security level
+* Enforce Response Type of OIDC Hybrid Flow including ID Token used as _detached signature_ as described in the FAPI 1 specification, which means that ID Token returned from Authorization response won't contain user profile data
+* Enforce more secure `state` and `nonce` parameters treatment for preventing CSRF
+* Enforce more secure signature algorithm when client registration
+
+[[_client_policy_profile]]
+===== Profile
+
+A profile consists of several executors which can realize a security profile like FAPI. Profile can be configured by the Admin REST API (admin console) together with it's executors.
+There are two _global profiles_ configured in the {project_name} by default with pre-configured executors compliant with the FAPI Baseline and FAPI Advanced specifications.
+More details in the FAPI section of the link:{adapterguide_link}#_fapi-support[{adapterguide_name}].
+
+[[_client_policy_policy]]
+===== Policy
+
+A policy consists of several conditions and profiles. The policy can be adopted to clients satisfying any condition of this policy. The policy refers several profiles and all
+executors of these profiles execute their task against the client that this policy is adopted.
+
+
+==== Configuration
+
+Policies, profiles, conditions, executors can be configured by Admin REST API, which means also an admin console. To do so, there is a tab _Realm_ -> _Realm Settings_ -> _Client Policies_
+, which means an administrator can have client policies per realm.
+
+The _Global Client Profiles_ are automatically available in each realm. However there are not client policies
+configured by default. This means that administrator is always required to create any client policy if he wants for example the clients of his realm to be FAPI compliant. Global
```

Comment:
"he wants?"  What if the administrator is a woman?
Try this "For example, if you want the clients of your realm to be FAPI compliant, a client policy is required"